pkg/smartcontract/parameter.go

```diff
@@ -152,10 +163,19 @@ func (p *Parameter) UnmarshalJSON(data []byte) (err error) {
 		if err = json.Unmarshal(r.Value, &s); err != nil {
 			return
 		}
-		if i, err = strconv.ParseInt(s, 10, 64); err != nil {
+		i, err = strconv.ParseInt(s, 10, 64)
+		if err == nil {
+			p.Value = i
 			return
 		}
-		p.Value = i
+
+		bi, ok := new(big.Int).SetString(s, 10)
```

Comment:
Maybe we can use `big.Int` always? It's only used by RPC client and CLI, it's not performance-critical.